source/_posts/2017/2017-01-10-jak-psat-zajimave-inzeraty-pro-ajtaky.md

```diff
@@ -0,0 +1,183 @@
+'''
+reviewed_by: [3, 4]
+---
+
+
+## Čím začneme?
+
+### Inzerát je reklamní článek, měl by vypadat jako od copywritera
+
+Celý inzerát je jedno svádění. Na konci musíte kandidáta dostat do postele, tak aby se mu to líbilo. Až tak, že vám druhý den ráno napíše. 
+ 
+## Jak na to? Není to o vás, je to o tom, kdo hledá
+
+Spousta inzerátu je psáno spíše jako sebeprezentace firmy - co umí, v čem je nejlepší a co dokázala. **Lidi čtou, protože chtějí něco pro sebe** - informace, peníze nebo zábavu. **Co ze článku nebo pracovní pozice budou mít, hledají hned na začátku. Dejte jim to do první třetiny!**
+
+Tip na to, jak lépe psát najdeš tady: 
+[3 minižíky od Otty](http://ottocopy.cz/ebook-zdarma) - hledej sekci *O nás*.
+
+### Buďte upřímní
+
+Uvádějte jen technologie, které opravdu máte. Ne ty, které plánujte nebo chcete, ale ty které opravdu máte. Když vám na pohovor přijde člověk, který se těší na PHP 7, a vy ho budete teprve nasazovat (kdo ví kdy), začne mít pocit, že vaříte z vody.
+
+**Text musí být upřímný, aby čtenář dostal to, co očekává. Jedině tak vám bude věřit a vrátí se k vám.**
+
+### Kamarád taky rád
+
+**Když u vás pracuje někdo šikovný, zmiňte ho v inzerátu**. Lidé půjdou spíše za kamarádem nebo někým v komunitě známým, než jen do firmy.
+
+Sám si pamatuju tyhle dvojice (i když už tam někteří nepracují): 
+
+- Filip Procházka - DámeJídlo, Rohlík 
+- Ondrej Mirtes - Slevomat
+- Patrik Votoček - Shipito
+- Vašek Purchart - Lavito
+
+
+### Řeknete si o feedback - dovnitř i ven
+
+**1. Řekněte si o feedback v práci** 
+
+„Co tě nakonec dovedlo k nám místo ke konkurenci?“
+
+**2. Dejte to přečíst někomu zvenčí**
+
+Vaši zaměstnanci vás přirozeně vidí lépe než někdo, koho neživíte. Lidé zvenčí vám dají upřímnější zpětnou vazbu a spíše řeknou, co potřebují pro navázání vztahu.
+ 
+**3. Které dvě věci se ti nejvíc líbí a které dvě bys zlepšil?**    
+    
+Napište 3 lidem z cílovky. Tak máte šanci se dozvědět, jak působíte navenek &ndash; očima člověka, kterého hledáte.
+
+       
+### Svlékněte se
+
+Ukažte, jak to doopravdy děláte. Důkaz místo slibů.
+
+**Děláte open-source?** Ven s ním. Vím, že Skrz, Jobs, Slevomat, PeckaDesign nebo Shopsys mají balíčky na Githubu. Bohužel, ani u jednoho z inzerátů jsem odkaz nenašel.
+
+Více informací mi může dát jenom náhled do zdrojáků aplikace. **Open-source je velmi podceňovaný hiring nástroj.**
+    
+### Odlište se
+
+Prodejte své odlišnosti. Proto k vám lidi jdou, ne protože stejně jako ostatní děláte v PHP, Symfony, Nette a Doctrine. Taková informace by tam měla být, ale určitě ne na předních místech.
+
+### Příklady přitažlivých odlišností
```

Comment:
Mně se líbí skrz tím, že dělají cirkusy a sdílejí i části svého know-how ven. Tím na mě, jakožto vývojáře, potom tato firma působí velmi atraktivně.